base/dates/accessors.jl

```diff
@@ -89,3 +72,68 @@ yearmonthday(dt::TimeType) = yearmonthday(days(dt))
 @vectorize_1arg TimeType yearmonth
 @vectorize_1arg TimeType monthday
 @vectorize_1arg TimeType yearmonthday
+
+
+# Documentation for exported accessors
+for func in (:year, :month)
+    name = string(func)
+    @eval begin
+        @doc """
+            $($name)(dt::TimeType) -> Int64
+
+        The $($name) of a `Date` or `DateTime` as a `Int64`.
+        """ $func(dt::TimeType)
+    end
+end
+
+"""
+    week(dt::TimeType) -> Int64
```

Comment:
I was wondering about this issue myself. The manual currently groups all of these accessors together like this:

```
    year(dt::TimeType) -> Int64
    month(dt::TimeType) -> Int64
    week(dt::TimeType) -> Int64
    day(dt::TimeType) -> Int64
    hour(dt::TimeType) -> Int64
    minute(dt::TimeType) -> Int64
    second(dt::TimeType) -> Int64
    millisecond(dt::TimeType) -> Int64

Return the field part of a `Date` or `DateTime` as an `Int64`.
```
I somewhat like how this looks in the manual as it combines what can be redundant documentation. Unfortunately details like what I have added for `week` don't make it into the manual and are only available via the REPL.